setup

```diff
@@ -0,0 +1,212 @@
+import subprocess
+
+DEFAULT_INPUTS = {
+    "BETA_OPT_IN": {
+        "default": "No",
+        "skip": False,
+        "help": "Type in 'yes' to opt-in for beta."
+    },
+    "CORE_TAG": {
+        "default": "crestsystems/cloudexchange:core-latest",
+        "skip": True,
+        "help": ""
+    },
+    "UI_TAG": {
+        "default": "crestsystems/cloudexchange:ui-latest",
+        "skip": True,
+        "help": ""
+    },
+    "UI_PORT": {
+        "default": 80,
+        "skip": False,
+        "help": ""
+    },
+    "JWT_SECRET": {
+        "default": "secret",
+        "skip": False,
+        "help": ""
+    },
+    "WATCHTOWER_TOKEN": {
+        "default": "token",
+        "skip": True,
+        "help": ""
+    },
+    "MONGODB_USERNAME": {
+        "default": "cteadmin",
+        "skip": True,
+        "help": ""
+    },
+    "MONGODB_PASSWORD": {
+        "default": "cteadmin",
+        "skip": True,
+        "help": ""
+    },
+    "DOCKER_USERNAME": {
+        "default": "",
+        "skip": True,
+        "help": ""
+    },
+    "DOCKER_PASSWORD": {
+        "default": "",
+        "skip": True,
+        "help": ""
+    },
+    "MAX_MAINTENANCE_WINDOW_MINUTES": {
+        "default": 15,
+        "skip": True,
+        "help": ""
+    },
+    "PULL_THREADS": {
+        "default": 6,
+        "skip": True,
+        "help": ""
+    },
+    "MAX_WAIT_ON_LOCK_IN_MINUTES": {
+        "default": 240,
+        "skip": True,
+        "help": ""
+    },
+}
+
+AVAILABLE_INPUTS = {}
+
+CORE_DEFAULT = "crestsystems/cloudexchange:core-latest"
+UI_DEFAULT = "crestsystems/cloudexchange:ui-latest"
+CORE_BETA_DEFAULT = "crestsystems/cloudexchange:core-latest-beta"
+UI_BETA_DEFAULT = "crestsystems/cloudexchange:ui-latest-beta"
+
+
+def put_env_variable(inputs):
+    try:
+        with open(".env", "w") as f:
+            for key, value in inputs.items():
+                f.write(f"{key}={value}\n")
+    except Exception as e:
+        raise Exception(f"Error occured while putting env variables: {e}")
+
+
+def create_env_if_not_exist():
+    try:
+        with open(".env", "a") as f:
+            pass
+    except Exception as e:
+        raise Exception(f"Error occured while creating file: {e}")
+
+def get_all_existed_env_variable():
+    try:
+        with open(".env", "r") as f:
+            for line in f.readlines():
+                key, value = line.split('=')
+                AVAILABLE_INPUTS[key] = value.strip()
+    except Exception as e:
+        raise Exception(f"Error occured while getting env variables: {e}")
+
+
+def set_directory_permission(directory, command):
+    try:
+        print(f"\nSetting permission for {directory} folder...")
+        command = "sudo chown -R 1001:1001 data/mongo-data"
+        p = subprocess.Popen(command.split(), stderr=subprocess.PIPE, stdout=subprocess.PIPE,  stdin=subprocess.PIPE) 
+        out, err = p.communicate()
+        if len(err) <= 0:
+            print("Permission set Successfully !!!")
```

Comment:
Update message to "Permission set successfully...".

Reply:
Updated